views/404error.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
Is `errorPage.html` supposed to be deleted?

Reply:
yeah it was supposed to be deleted, it was accidently included in an earlier commit and is unused